### ISSUE TITLE
SA1652 is incorrectly triggered in Visual Studio if <DocumentationFile> is only defined in an imported .targets file.

### ISSUE THREAD

**Issue by stylpe** [DotNetAnalyzers/StyleCopAnalyzers#1572]
Somewhere in Project.csproj:

```xml
<Import Project="$(SolutionDir)\lib\BuildTools\Common.Targets">
```

Common.Targets (minimal):

```xml
<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
  <PropertyGroup>
    <DocumentationFile>$(OutputPath)$(AssemblyName).xml</DocumentationFile>
  </PropertyGroup>
</Project>
```

Reproducing has been a bit inconsistent. They sometimes temporarily disappear through cleaning/rebuilding individual projects in the solution, but they're back as soon as I close and reopen the solution.

Building with msbuild does not trigger this.

**Comment by sharwell** [comment]
I'm guessing this is a bug in Roslyn. I filed dotnet/roslyn#5525.

**Comment by sharwell** [comment]
@stylpe Can you take a look at dotnet/roslyn#5525? The team there is asking for help in reproducing the behavior you saw.

**Comment by stylpe** [comment]
It's due to caching at some level in VS. I found the solution: Completely restart VS. I had only tried clesn, rebuild, close and open solution. I'll close this issue (SA1652 is gone after restarting) and provide more details in dotnet/roslyn#5525 (but I kinda doubt that Roslyn is the culprit)
